getting-started.md

```diff
@@ -72,13 +70,13 @@ If everything works, you should see your starter app on your device or simulator
 <div id="starter-app-screenshots">
   <div class="box2">
     <figure>
-      <img src="/images/flutter-starter-app-ios.png">
+      <img src="/images/flutter-starter-app-ios.png" alt="iOS screenshot">
```

Comment:
that is not a valid alt value. It is more correct to omit the alt attribute here.